### ISSUE TITLE
Add Doc on using Android Emulator

### ISSUE THREAD

**Issue by adammichaelwood** [opendatakit/docs#289]
Is this worth pulling in? 
Or is it really only helpful to ODK developers?

https://github.com/opendatakit/opendatakit/wiki/DevEnv-Setup

**Comment by yanokwa** [comment]
I think this is useful to implementers and those folks can't always figure out Android's obtuse instructions. I think something more step-by-step here would save a lot of people a lot of time. 

Related: https://github.com/opendatakit/docs/issues/306 https://github.com/opendatakit/docs/issues/307

**Comment by ankita240796** [comment]
@adammichaelwood May I work on this.

**Comment by adammichaelwood** [comment]
Is there anything still needed after @Divya063's work in #327:
http://docs.opendatakit.org/projecting-collect/#using-android-studio

That seems to cover things (though a review/cross-check with the wiki page is worth doing). 
I do wonder if the emulator stuff should be put in its own doc, as it is useful outside of projecting on-screen. Then the projecting page could link to it.

Thoughts?

**Comment by Divya063** [comment]
@adammichaelwood The information I added in #327 covers everything, there is only one thing left  i.e.  adding the command line option.

**Comment by ankita240796** [comment]
Yes, @Divya063's work covers everything but I too feel that putting emulator stuff in its own doc is a good idea as it will be useful to implementers. In the present scenario, it is represented as a sub-section in projecting on-screen docs but it is better to have a doc which describes it in general and the projecting on-screen doc can then be linked to it.

**Comment by adammichaelwood** [comment]
@Divya063 
Since it was your work -- if you have time -- why don't you pull that into it's own doc and link as suggested, plus add the command line thing you mentioned.

**Comment by Divya063** [comment]
@adammichaelwood Yes, I would start working on this, so should I add the [deprecated method](https://github.com/opendatakit/docs/issues/306#issuecomment-339021832) too? 
As the previous versions of SDK are still accessible, this method can be still adopted.

**Comment by adammichaelwood** [comment]
I think we probably shouldn't.

**Comment by Divya063** [comment]
@adammichaelwood The stand-alone SDK package provided on the [website](https://developer.android.com/studio/index.html) does not contain [**Quick Emulator**](http://wiki.qemu.org/) and some necessary folders which are required for running an emulator. So, users have to download Android Studio along with SDK  in order to run the emulator from the command line. I have created a zip file which contains all the necessary folders including QEMU needed to run the emulator without using Android studio. Maybe we can provide the zip file to the users or we can just provide instructions on how to run emulator using the command line (assuming that user has downloaded Android studio).

**Comment by adammichaelwood** [comment]
> we can just provide instructions on how to run emulator using the command line (assuming that user has downloaded Android studio).

This sounds reasonable.

> Maybe we can provide the zip file to the users 

I think providing third-party software to docs users is not a prudent idea.

**Comment by Divya063** [comment]
@adammichaelwood Yes, that would be reasonable. I'll add instructions on that.
